### PR TITLE
Update to 18.04 and PX4 v1.9

### PR DESCRIPTION
**Pull request by julianoes**
This is an attempt to use this docker image for the Dronecode SDK integration tests.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,32 +18,21 @@ RUN apt-get update && \
...
-RUN chmod +x /tmp/install_gazebosim.sh
-RUN /tmp/install_gazebosim.sh
+                       unzip \
+                       gazebo9 \
+                       libgazebo9-dev
 
 RUN git clone --depth 1 https://github.com/PX4/Firmware.git ${FIRMWARE_DIR}
 RUN git -C ${FIRMWARE_DIR} submodule update --init --recursive
 
-# Initial location of the drone
```
Comment (JonasVautherin) [inline Dockerfile]: Why removing those?

File: Dockerfile [hunk 2]
```diff
@@ -1,8 +1,10 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 
 ENV WORKSPACE_DIR /root
 ENV FIRMWARE_DIR ${WORKSPACE_DIR}/Firmware
 
+ENV DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true
```
Comment (JonasVautherin) [inline Dockerfile]: Why is that required?

File: Dockerfile [hunk 3]
```diff
@@ -16,32 +18,21 @@ RUN apt-get update && \
...
                        python-toml \
                        python-yaml \
-                       unzip
-
-RUN curl -ssL http://get.gazebosim.org > /tmp/install_gazebosim.sh
-RUN sed -i "s/GZ_VER=9/GZ_VER=8/" /tmp/install_gazebosim.sh
-RUN chmod +x /tmp/install_gazebosim.sh
-RUN /tmp/install_gazebosim.sh
+                       unzip \
+                       gazebo9 \
```
Comment (JonasVautherin) [inline Dockerfile]: Yes it's nicer like this!

File: Dockerfile (hunk 1 above)
Comment (julianoes) [reply Dockerfile]: See commits messages.

File: Dockerfile (hunk 2 above)
Comment (julianoes) [reply Dockerfile]: See commit messages.

File: Dockerfile (hunk 3 above)
Comment (julianoes) [reply Dockerfile]: Yes that was easy.

File: Dockerfile (hunk 1 above)
Comment (JonasVautherin) [reply Dockerfile]: Argh, I always do that and rely on the PR description :sweat_smile:

File: Dockerfile (hunk 3 above)
Comment (JonasVautherin) [reply Dockerfile]: Until we move to gazebo10 and stay on Ubuntu 18 :sweat_smile:

**Comment by julianoes** [comment]
I can't actually merge this, it's up to you!

**Comment by JonasVautherin** [comment]
Oh sorry :sweat_smile:
